.agent/specs/324.md

```
@@ -0,0 +1,541 @@
+### BE API (모두 구현 완료)
+
+| Method | Path | 설명 | 응답 타입 |
+|--------|------|------|-----------|
+| GET | `.../workflows` | 목록 조회 | `WorkflowSummary[]` |
+| GET | `.../workflows/{workflowId}` | 단건 조회 | `WorkflowDefinitionDetail` |
+| PATCH | `.../workflows/{workflowId}` | 필드 + 그래프 수정 | `WorkflowDefinitionDetail` |
+
+공통 path prefix: `/api/v1/workspaces/{workspaceId}/domain-packs/{packId}/versions/{versionId}`
+
+PATCH request body:
+```json
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)